src/content/community/releases/_index.en.md

```diff
@@ -5,6 +5,13 @@ weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
 
+## v0.15.1
+
+This is a bugfix release:
+
+* Fixed IPsec pluto crash when remote endpoint is unstable.
```

Comment:
To be consistent with https://github.com/submariner-io/submariner-website/pull/944/files.

```suggestion
* Fixed issue which caused the IPsec pluto process to crash when the remote endpoint was unstable.
```